pygmt/tests/test_grdinfo.py

```diff
@@ -13,15 +13,15 @@ def test_grdinfo():
     Make sure grd info works as expected.
     """
     grid = load_earth_relief(registration="gridline")
-    result = grdinfo(grid, L=0, C="n")
+    result = grdinfo(grid=grid, L=0, C="n")
     assert result.strip() == "-180 180 -90 90 -8592.5 5559 1 1 361 181 0 1"
```

Comment:
Could you please change all the short-form arguments (e.g., `L`) to their long-form aliases (e.g., `force_scan`)?

Reply:
Changed in c47d45a